src/ClientData/ModuleManager.cpp

```diff
@@ -67,6 +68,40 @@ std::vector<ModuleData*> ModuleManager::AddOrUpdateNotLoadedModules(
   return not_updated_modules;
 }
 
+const ModuleData* ModuleManager::GetModuleByModuleInMemoryAndAbsoluteAddress(
+    const ModuleInMemory& module_in_memory, uint64_t absolute_address) const {
+  absl::MutexLock lock(&mutex_);
```

Comment:
We should avoid code duplication and call `GetMutableModuleByModuleInMemoryAndAbsoluteAddress` here, wdyt?

Reply:
I do not know how to do that without const_cast<>, If you call non-const method from this one - you need to const_cast this, if you deside to call const from non-const method you will need to const_cast the result.

Reply:
Right, the method itself is also non-const....